docs/docsite/_themes/sphinx_rtd_theme/ansible_banner.html

```diff
@@ -4,7 +4,7 @@
     if (location.host == "docs.testing.ansible.com") {
       document.write('<div id="testing_banner_id" class="admonition important">');
       para = document.createElement('p');
-      banner_text=document.createTextNode("This is a testing site. For the official docs go to https://docs.ansible.com/");
+      banner_text=document.innerHTML('<b>This is the testing site for Ansible Documentation<b/>. For the official documentation website visit <a href="https://docs.ansible.com/">https://docs.ansible.com/</a>');
```

Comment:
Should the end-tag be `</b>` instead of `<b/>`?

Reply:
indeed